web/src/routes/admin/system-settings/+page.svelte

```diff
@@ -68,104 +90,128 @@
     title: string;
     subtitle: string;
     key: string;
+    icon: string;
   }> = [
     {
       component: AuthSettings,
       title: $t('admin.authentication_settings'),
       subtitle: $t('admin.authentication_settings_description'),
-      key: 'image',
+      key: 'authentication',
+      icon: mdiLockOutline,
     },
     {
       component: ImageSettings,
       title: $t('admin.image_settings'),
       subtitle: $t('admin.image_settings_description'),
       key: 'image',
+      icon: mdiImageOutline,
     },
     {
       component: JobSettings,
       title: $t('admin.job_settings'),
       subtitle: $t('admin.job_settings_description'),
       key: 'job',
+      icon: mdiSync,
     },
     {
       component: MetadataSettings,
       title: $t('admin.metadata_settings'),
       subtitle: $t('admin.metadata_settings_description'),
       key: 'metadata',
+      icon: mdiDatabaseOutline,
     },
     {
       component: LibrarySettings,
       title: $t('admin.library_settings'),
       subtitle: $t('admin.library_settings_description'),
       key: 'external-library',
+      icon: mdiLibraryOutline,
```

Comment:
I think this should match the icon in the sidebar.

```suggestion
      icon: mdiBookshelf,
```